### PR TITLE
Initialise matcher functions

### PR DESCRIPTION
**Pull request by vn09**

### CODE REVIEW

**Comment by ca7erina** [comment]
getMatchedUsers
getWaitingForUsers
getNewSuggestionUsers

File: app/models/User.java [hunk 1]
```diff
@@ -67,6 +70,17 @@ public static User findByFullname(String fullname) {
...
+     *
+     * @param email email to search
+     * @return List of users
+     */
+    public static List<User> findSimilarityFullname(String email) {
+        String fullname = User.findByEmail(email).fullname;
+        return find.where().ilike("fullname", "%" + fullname + "%").setMaxRows(User.MAX_LIMIT_ROWS).findList();
+    }
+
+    /**
```
Comment (vn09) [inline app/models/User.java]: Add mock findSimilarityFullname for testing

File: conf/evolutions/default/2.sql [hunk 2]
```diff
@@ -24,6 +24,27 @@ VALUES
     NOW(),
     1),
     (user_seq.nextval,
+    'admin2@gmail.com',
```
Comment (vn09) [inline conf/evolutions/default/2.sql]: Add more users that have similar fullname for testing

File: test/controllers/ApplicationTest.java [hunk 3]
```diff
@@ -15,21 +15,21 @@
 /**
  * Tests for Login
  */
-public class LoginTest extends WithApplication {
+public class ApplicationTest extends WithApplication {
```
Comment (vn09) [inline test/controllers/ApplicationTest.java]: refactor test codes

**Pull request merged**